examples/src/main/java/ExampleCreateVoiceCallFlow.java

```diff
@@ -0,0 +1,43 @@
+import com.messagebird.MessageBirdClient;
+import com.messagebird.MessageBirdService;
+import com.messagebird.MessageBirdServiceImpl;
+import com.messagebird.exceptions.GeneralException;
+import com.messagebird.exceptions.UnauthorizedException;
+import com.messagebird.objects.voicecalls.VoiceCallFlowRequest;
+import com.messagebird.objects.VoiceStep;
+
+import java.util.Collections;
+
+public class ExampleCreateVoiceCallFlow {
+
+    public static void main(String[] args) {
+        if (args.length < 4) {
+            System.out.println("Please specify your access key and voice call flow arguments");
```

Comment:
why didn't you use args[1], args[3] but you asked for them

Reply:
I refactored the example a little and forgot about these arguments. It should look better now